gatsby-node.js

```diff
@@ -42,25 +42,33 @@
       }
     }
     `).then(result => {
-     result.data.allShopifyProduct.edges.forEach(({ node }) => {
-       createPage({
-         path: `product/${node.id}`,
-         component: path.resolve(`./src/pages/demoProductItem.js`),
-         context: {
-           productId: node.id,
-           productName: node.title,
-           images: node.images,
-           tags: node.tags,
-           productPrice: node.priceRange.minVariantPrice.amount,
-           variants: node.variants,
-           description: node.description
-         },
-       })
-     })
-     resolve()
-     })
-   }).catch(error => {
-     console.log(error)
-     reject()
-   })
- };
\ No newline at end of file
+        result.data.allShopifyProduct.edges.forEach(({ node }) => {
```

Comment:
We are fetching and iterating the same data here and again in `demoCategories.js` page for each category.

Instead of fetching again in `demoCategories.js`, can we pass products under each category as props to that page?

Reply:
Yes can be done. Will updated the code soon.